rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -1163,6 +1164,18 @@ stream {
             {{- end }}
             {{ end }}
 
+            {{ if not (isLocationInLocationList $location $all.Cfg.NoAuthLocations) }}
```

Comment:
What if both auth-url annotation and global setting is set? If I read this correctly it'll have duplicate auth request setup. We should make sure annotation overrides global setting.

Reply:
Okay, can do it. Was not sure about the priorities. Now it is clear.